strapi/strapi
# CellContent: Refactor relations to use the SimpleMenu component

### What does it do?

It unifies the visual style with components shown in the list view and re-uses the `SimpleComponent` from the design system. The necessary `onOpen` and `onClose` callbacks were introduced in https://github.com/strapi/design-system/pull/545. This PR waits for the design system 1.0.0 release.

<img width="1157" alt="Screenshot 2022-02-23 at 15 03 47" src="https://user-images.githubusercontent.com/2244375/155334689-23c0d4a7-36cd-4124-abd9-fcca5c395fef.png">

### Why is it needed?

To sync the visual style of components and relations in the list view.

### How to test it?

1. Create a content type with a relation
2. Display that relation in the content manager list view
3. Make sure it loads it's content dynamically

### Related issue(s)/PR(s)

- https://github.com/strapi/strapi/pull/12213
- https://github.com/strapi/design-system/pull/545
